### PR TITLE
Update policy-csp-update.md

### PR DESCRIPTION
**Pull request by itsrlyAria**
This is correcting the description of AU Options.

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-update.md [hunk 1]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
--   4 – Auto install and restart without end-user control. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates right away. If a restart is required, then the device is automatically restarted when the device is not actively being used. This setting option also sets the end-user control panel to read-only.
+-   4 – Auto install and restart at a specified time. The IT specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart. Note, this option is the same as 3, but restricts end user controls on the settings page.
```
Comment (illfated) [inline windows/client-management/mdm/policy-csp-update.md:483]: Could we add "admin" to "The IT", for consistency and clarity throughout this document?
```suggestion
-   4 – Auto install and restart at a specified time. The IT Admin specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart. Note, this option is the same as 3, but restricts end user controls on the settings page. 
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 2]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
+-   2 – Auto install and restart. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates right away. If a restart is required, then the device is automatically restarted when the device is not actively being used. This is the default behavior for unmanaged devices. Devices are updated quickly, but it increases the risk of accidental data loss caused by an application that does not shutdown properly on restart.
 -   3 – Auto install and restart at a specified time. The IT specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart.
```
Comment (illfated) [inline windows/client-management/mdm/policy-csp-update.md:482]: As in my previous suggestion, even though this is not part of your changes, may we have the same change here?
```suggestion
-   3 – Auto install and restart at a specified time. The IT Admin specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart.
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 3]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
+-   1 – Auto install the update and then notify the user to schedule a device restart. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates immediately. If the installation requires a restart, the end-user is prompted to schedule the restart. The end-user has up to seven days to schedule the restart and after that, a restart of the device is forced. Enabling the end-user to control the start time reduces the risk of accidental data loss caused by applications that do not shutdown properly on restart.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:480]:
```suggestion
-   1 – Auto install the update and then notify the user to schedule a device restart. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates immediately. If the installation requires a restart, the end-user is prompted to schedule the restart. The end-user has up to seven days to schedule the restart and after that, a restart of the device is forced. Enabling the end-user to control the start time reduces the risk of accidental data loss caused by applications that do not shut down properly on restart.
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 4]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
+-   2 – Auto install and restart. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates right away. If a restart is required, then the device is automatically restarted when the device is not actively being used. This is the default behavior for unmanaged devices. Devices are updated quickly, but it increases the risk of accidental data loss caused by an application that does not shutdown properly on restart.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:481]:
```suggestion
-   2 – Auto install and restart. Updates are downloaded automatically on non-metered networks and installed during "Automatic Maintenance" when the device is not in use and is not running on battery power. If automatic maintenance is unable to install updates for two days, Windows Update will install updates right away. If a restart is required, then the device is automatically restarted when the device is not actively being used. This is the default behavior for unmanaged devices. Devices are updated quickly, but it increases the risk of accidental data loss caused by an application that does not shut down properly on restart.
```

File: windows/client-management/mdm/policy-csp-update.md [hunk 5]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
+-   3 – Auto install and restart at a specified time. The IT Admin specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:482]:
```suggestion
-   3 – Auto install and restart at a specified time. The IT Admin specifies the installation day and time. If no day and time is specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@itsrlyAria - although my suggestions concern only text carried over from the existing copy, could you please commit my suggested changes? I have also included @illfated's suggestion. Thanks.

File: windows/client-management/mdm/policy-csp-update.md [hunk 6]
```diff
@@ -477,11 +477,12 @@ ADMX Info:
...
+-   4 – Auto install and restart at a specified time. The IT specifies the installation day and time. If no day and time are specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart. Note, this option is the same as 3, but restricts end user controls on the settings page.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-update.md:483]:
```suggestion
-   4 – Auto install and restart at a specified time. The IT Admin specifies the installation day and time. If no day and time is specified, the default is 3 AM daily. Automatic installation happens at this time and device restart happens after a 15-minute countdown. If the user is logged in when Windows is ready to restart, the user can interrupt the 15-minute countdown to delay the restart. Note, this option is the same as 3, but restricts end-user controls on the settings page. 
```

**Comment by e0i** [comment]
Hello, @itsrlyAria and welcome to the IT Pro repository. You need to apply the copy editing suggestions provided by reviewers in order to proceed with final review and merge. Please feel free to re-open this PR whenever you are able to do so.  Thank you for your very first contribution, and stay healthy.

**Pull request closed without merging**